src/layouts/partials/NotificationContent/NotificationContent.jsx

```diff
@@ -50,18 +58,60 @@ class NotificationContent extends PureComponent {
     this.props.onClose()
   }
 
-  renderTransaction (trx) {
-    const hash = trx.hash()
-    const details = trx.details()
+  getCurrentTransactionNotificationList = () => {
+    const { ethTransactionsList, btcTransactionsList } = this.props
+    const list = []
+
+    ethTransactionsList.map((item) => {
+      list.push(this.convertToCurrentTransactionNotification(item))
+    })
+    btcTransactionsList.map((item) => {
+      list.push(this.convertToCurrentTransactionNotification(item))
+    })
+
+    return list
+  }
+
+  convertToCurrentTransactionNotification (transaction) {
+    switch (true) {
+      // Eth transactions
+      case transaction instanceof TxExecModel:
+        return new CurrentTransactionNotificationModel({
+          id: transaction.hash(),
+          title: transaction.title(),
+          date: transaction.time(),
+          details: transaction.details(),
+        })
+
+      // BTC transactions
+      case transaction instanceof TxModel:
+        return new CurrentTransactionNotificationModel({
+          id: transaction.txHash(),
+          title: `${transaction.symbol()} Transfer`,
+          date: transaction.time(),
+          details: transaction.details(),
+        })
+
+      default:
+        break
+
+    }
+
+  }
+
+  renderTransaction (notification: CurrentTransactionNotificationModel) {
+
+    const hash = notification.hash
```

Comment:
[MINOR]
I suggest to write something like this:
```javascript
const {
  hash,
  details,
  id,
  title
} = notification
```
and use these constants in the JSX below